core-services/prow/02_config/_config.yaml

```diff
@@ -101,6 +101,7 @@ deck:
           - Step .* failed after
           - ^make.*:.*\*\*\*
           - '[Ee]xit.*\s(code|status)\b.*\s[1-9][0-9]*\b'
+          - '│ Error:'
```

Comment:
reference for such line: https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-terraform-redhat-terraform-provider-ocm-main-managed-oidc-config-e2e-periodic/1667293554247995392#1:build-log.txt%3A100875